dbms/src/Storages/Page/V3/Universal/UniversalPageStorageService.cpp

```diff
@@ -26,19 +38,118 @@ UniversalPageStorageServicePtr UniversalPageStorageService::create(
     auto service = UniversalPageStorageServicePtr(new UniversalPageStorageService(context));
     service->uni_page_storage = UniversalPageStorage::create(name, delegator, config, context.getFileProvider());
     service->uni_page_storage->restore();
-    service->gc_handle = context.getBackgroundPool().addTask(
+    auto & bkg_pool = context.getBackgroundPool();
```

Comment:
I think we need a standalone thread to do only this job.

Reply:
create a standalone thread pool for it